nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Implement category list

# Description
Created a view to list all available categories. This will eventually be admin-only. Added a menu option for "Category Management" under "Admin Tools" on the main menu.
"Edit" and "Delete" buttons are present for each category, but are not yet functional.
Fixes #9 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Run SQL scripts to generate seed data
2. Run the program through VS debugger or CLI ```dotnet run```
3. Log in using ```admin@example.com```
4. Navigate to Admin Tools > Category Management
5. You should see the list of all available categories
# Checklist:
- [ ] My changes generate no new warnings or errors - Edit and Delete buttons will error out
